spec/services/employee_finder_spec.rb

```diff
@@ -0,0 +1,35 @@
+require "rails_helper"
+
+describe EmployeeFinder do
+  describe "#employee_exists?" do
+    it "returns true if an employee exists in Slack" do
```

Comment:
how about `it "returns true if an employee exists in the Slack organization associated with the auth token" do`

it's kinda wordy, but will be helpful for future people reading these tests!